src/axelrod_fortran/player.py

```diff
@@ -72,6 +75,18 @@ def original_strategy(
         return self.original_function(*[byref(arg) for arg in args])
 
     def strategy(self, opponent):
+        if type(opponent) is Player\
```

Comment:
space after player please, indentation looks like it's off by one character (but maybe not)